src/core/Dropdown/Dropdown.baseStyles.tsx

```diff
@@ -9,6 +9,7 @@ export const baseStyles = withSuomifiTheme(
       position: relative;
       padding-right: 30px;
       text-align: left;
+      background-color: ${theme.colors.whiteBase};
```

Comment:
Any idea where the grey background is coming from? Are there bleeding styles somewhere that could affect other things as well?

Reply:
At this point we dont wanna debug default styles of 3rd party lib